src/flag_gems/runtime/backend/_ascend/ops/angle.py

```diff
@@ -5,7 +5,8 @@
 import triton
 import triton.language as tl
 
-from flag_gems.utils import pointwise_dynamic, tl_extra_shim
+from ..utils.pointwise_dynamic import pointwise_dynamic

```

Comment:
can we do the following instead?
import from relative path is not a good practice IMHO.

```python
from flag_gems.utils import pointwise_dynamic
```